packages/ui/__tests__/Table/Table.test.tsx

```diff
@@ -168,7 +138,7 @@ describe('Table', () => {
       numberOfItems: smallDataSet.length,
     })
 
-    expect(wrapper.findDataTest('table-noData-cell').exists()).toBe(false)
+    expect(wrapper.find(EmptyState).exists()).toBe(false)
```

Comment:
I suggest passing test id into EmptyState. This might be very useful for e2e tests

Reply:
Changed in https://github.com/hazelcast/hive/pull/314/commits/12a8332dd23235cd3260ce320326a2a30d3285fc